### PR TITLE
Minor touchups

### PR DESCRIPTION
**Pull request by Camotoy**

### CODE REVIEW

File: connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java [hunk 1]
```diff
@@ -415,10 +415,10 @@ public void connect(RemoteServer remoteServer) {
 
         UpdateAttributesPacket attributesPacket = new UpdateAttributesPacket();
         attributesPacket.setRuntimeEntityId(getPlayerEntity().getGeyserId());
-        List<AttributeData> attributes = new ArrayList<>();
         // Default move speed
         // Bedrock clients move very fast by default until they get an attribute packet correcting the speed
-        attributes.add(new AttributeData("minecraft:movement", 0.0f, 1024f, 0.1f, 0.1f));
+        List<AttributeData> attributes = Collections.singletonList(
+                new AttributeData("minecraft:movement", 0.0f, 1024f, 0.1f, 0.1f));
         attributesPacket.setAttributes(attributes);
```
Comment (rtm516) [inline connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java:422]: Why not just inline this?

File: connector/src/main/java/org/geysermc/connector/network/translators/java/world/JavaPlayEffectTranslator.java [hunk 2]
```diff
@@ -76,9 +75,8 @@ public void translate(ServerPlayEffectPacket packet, GeyserSession session) {
                 textPacket.setPlatformChatId("");
                 textPacket.setSourceName(null);
                 textPacket.setMessage("record.nowPlaying");
-                List<String> params = new ArrayList<>();
                 String recordString = "%item." + soundEvent.name().toLowerCase(Locale.ROOT) + ".desc";
-                params.add(LocaleUtils.getLocaleString(recordString, session.getLocale()));
+                List<String> params = Collections.singletonList(LocaleUtils.getLocaleString(recordString, session.getLocale()));
                 textPacket.setParameters(params);
```
Comment (rtm516) [inline connector/src/main/java/org/geysermc/connector/network/translators/java/world/JavaPlayEffectTranslator.java:80]: Same as above

**Review by rtm516 — CHANGES_REQUESTED** [review]

File: connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java (hunk 1 above)
Comment (Camotoy) [reply connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java:422]: Good point.

File: connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java [hunk 3]
```diff
@@ -415,11 +415,10 @@ public void connect(RemoteServer remoteServer) {
 
         UpdateAttributesPacket attributesPacket = new UpdateAttributesPacket();
         attributesPacket.setRuntimeEntityId(getPlayerEntity().getGeyserId());
-        List<AttributeData> attributes = new ArrayList<>();
         // Default move speed
         // Bedrock clients move very fast by default until they get an attribute packet correcting the speed
-        attributes.add(new AttributeData("minecraft:movement", 0.0f, 1024f, 0.1f, 0.1f));
-        attributesPacket.setAttributes(attributes);
+        attributesPacket.setAttributes(Collections.singletonList(
```
Comment (lukeeey) [inline connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java:420]: one line like below?

Comment (Camotoy) [reply connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java:420]: Too long?

**Review by Tim203 — APPROVED** [review]

**Pull request merged**